Dijkstra_Algorithm/Dijkstra_Algorithm.dart

```diff
@@ -0,0 +1,96 @@
+/*
+ Djikstra's algorithm (named after its discover, E.W. Dijkstra) solves the
+ problem of finding the shortest path from a point in a graph (the source)
+ to a destination.
+ It turns out that one can find the shortest paths from a given source to all
+ points in a graph in the same time, hence this problem is sometimes called
+ the single-source shortest paths problem.
+*/
+int mind(dist, visited , n)
+{
+
+int min = 9223372036854775807, min_index;
+
+for (var v = 0; v < n; v++) {
+  if ((visited[v] == false) && (dist[v] <= min)) {
+    min = dist[v];
+    min_index = v;
+  }
+}
+
+return min_index;
+}
+
+void printsol(dist , n)
+{
+  print('Vertex \t\t Distance from Source\n');
+
+  for (var i = 0; i < n; i++)
+    {
+      print('${i} \t\t ${dist[i]}\n');
+    }
+}
+
+void dijkstra(graph, src, n)
+{
+  var dist = new List(n);
+
+  var visited = new List(n);
+
+  for (var i = 0; i < n; i++) {
+    dist[i] = 9223372036854775807;
+    visited[i] = false;
+  }
+
+  dist[src] = 0;
+
+  for (var count = 0; count < n - 1; count++)
+  {
+
+    var u = mind(dist, visited , n);
+
+    visited[u] = true;
+
+    for (var v = 0; v < n; v++)
+    {
+        if (!visited[v] && graph[u][v]>0 && dist[u] != 9223372036854775807
```

Comment:
Please store the max value in some variable and use it, so use INT_MAX type of variable available in Dart.